kedro-exercises/spaceflight/src/kedro_tutorial/pipelines/data_engineering/__init__.py

```diff
@@ -1 +1,31 @@
+# Copyright 2020 QuantumBlack Visual Analytics Limited
```

Comment:
```suggestion
# Copyright 2021 QuantumBlack Visual Analytics Limited
```
I think you need to update this in all files.

Reply:
Done.